mkt/langpacks/models.py

```diff
@@ -66,6 +78,44 @@ def path_prefix(self):
     def file_path(self):
         return os.path.join(self.path_prefix, nfd_str(self.filename))
 
+    @property
+    def download_url(self):
+        url = ('%s/langpack.zip' %
+            reverse('downloads.langpack', args=[unicode(self.pk)]))
+        return absolutify(url)
+
+    @property
+    def manifest_url(self):
+        """Return URL to the minifest for the langpack"""
+        if self.active:
+            return absolutify(
+                reverse('langpack.manifest', args=[unicode(UUID(self.pk))]))
```

Comment:
Whats the advantage of using a uuid over a pk? And isn't the pk the incrementing id in the db?